lua/nvim-treesitter/parsers.lua

```diff
@@ -414,7 +432,8 @@ list.comment = {
 list.jsdoc = {
   install_info = {
     url = "https://github.com/tree-sitter/tree-sitter-jsdoc",
-    files = { "src/parser.c" }
+    files = { "src/parser.c" },
+    requires_generate_from_grammar  = true,
```

Comment:
jsdoc as well

Reply:
this one I forgot?

Reply:
Yes, the tag can be removed the version is already the newest.